plasmapy/formulary/collisions/tests/test_frequencies.py

```diff
@@ -661,6 +661,51 @@ def test_fundamental_frequency_values(
 
         assert np.allclose(calculated_value, expected_value, rtol=5e-3, atol=0)
 
+    @pytest.mark.parametrize(
+        ("frequency_to_test", "constructor_keyword_arguments", "expected_value"),
+        [
+            (
+                "Maxwellian_avg_ei_collision_freq",
+                {
+                    "test_particle": Particle("e-"),
+                    "field_particle": Particle("Na+"),
+                    "v_drift": 1 * u.m / u.s,
+                    "n_a": 1e26 * u.m**-3,
+                    "T_a": 1 * u.eV,
+                    "n_b": 1e26 * u.m**-3,
+                    "T_b": 1e3 * u.eV,
+                    "Coulomb_log": 10 * u.dimensionless_unscaled,
+                },
+                2.8053078e15 * u.Hz,
+            ),
+            (
+                "Maxwellian_avg_ii_collision_freq",
+                {
+                    "test_particle": Particle("Na+"),
+                    "field_particle": Particle("Na+"),
+                    "v_drift": 1 * u.m / u.s,
+                    "n_a": 1e26 * u.m**-3,
+                    "T_a": 1e3 * u.eV,
+                    "n_b": 1e26 * u.m**-3,
+                    "T_b": 1e3 * u.eV,
+                    "Coulomb_log": 10 * u.dimensionless_unscaled,
+                },
+                1.1223822e8 * u.Hz,
+            ),
+        ],
+    )
+    @pytest.mark.filterwarnings("ignore::plasmapy.utils.exceptions.RelativityWarning")
+    def test_correctness_collision_freq_values(
```

Comment:
@namurphy Isn't there a name for when you test software to see if it reproduces a previously recorded output? 

I'd suggest we change the name here to `test_collision_freq_values_match_previous_evaluation` or something, since `correctness` implies that the actual physics is being tested here? Unless these numbers are coming from a by-hand calculation, in which case this test is an independent verification of the physics in code?